GitHub health files (#206)
BioPhoton/rx-angular: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve

> title: "[State]: Title of the bug"

Reviewer: What is "[State]"?

Author: Some string helping to categorize the issue. 
We can remove it as we have labels :)

> title: "[State]: Title of the bug"

Author: ```suggestion
title: "Title of the bug"
```

> title: "[State]: Title of the bug"

Author: ```suggestion
title: "Title of the bug"
```

> labels: ''

Reviewer: ```suggestion
labels: 'state | template'
```
shouldn't we mention to use either state or template label? Wherever it fits best?

assignees: ''

---

# Description
<!--
describe expected vs actual behavior

if applicable and relevant include any of the following:

- screenshots
- stack trace
- filename and line number where the code breaks
- other debug output
- minimal code examples/test case that produces the issue (especially if it relates to library code)
- dom snippets
- JSON of the broken state
- etc
-->

# Steps to Reproduce the Issue
<!--

> Explain in detail hoe to reproduce the issue:
> 
> 1. Do Something
> 2. ???
> 3. Bug
> 

Reviewer: ```suggestion
Explain detailed information about the steps how to reproduce this specific bug

```
